logging/zerolog_adapter/event_builder.go

```diff
@@ -1,9 +1,9 @@
-package logging
+package zerolog_dapter
```

Comment:
```suggestion
package zerolog_adapter
```

Typo

Reply:
That typo also in other PRs. I think it is time to merge to `feature/refactor-logging-for-sentry` and fix conflicts

Reply:
It should be fixed in main PR. 

I will merge this brach the momoent you approve it.